examples/dt/nfv/nfv-ovs-dpdk-sriov-hci/dataplane-pre-ceph.md

```diff
@@ -0,0 +1,50 @@
+# Configuring and deploying the pre-Ceph dataplane
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully deployed
+
+## Initialize pre-Ceph
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the hci directory
+```
+cd architecture/examples/dt/nfv/nfv-ovs-dpdk-sriov-hci
+```
+Edit the [edpm-pre-ceph/nodeset/values.yaml](edpm-pre-ceph/nodeset/values.yaml) file to suit
+your environment.
+```
+vi edpm-pre-ceph/nodeset/values.yaml
+```
+Generate the pre-Ceph dataplane nodeset CR.
+```
+kustomize build edpm-pre-ceph/nodeset > dataplane-nodeset-pre-ceph.yaml
+```
+Generate the pre-Ceph dataplane deployment CR.
+```
+kustomize build edpm-pre-ceph/deployment > dataplane-deployment-pre-ceph.yaml
```

Comment:
Very nice! I see this is now producing a deployment with the correct `servicesOverride:`

```
[fultonj@runcible nfv-ovs-dpdk-sriov-hci{nfvhci}]$ kustomize build edpm-pre-ceph/deployment
apiVersion: dataplane.openstack.org/v1beta1
kind: OpenStackDataPlaneDeployment
metadata:
  name: edpm-deployment
  namespace: openstack
spec:
  nodeSets:
  - openstack-edpm
  servicesOverride:
  - bootstrap
  - download-cache
  - reboot-os
  - configure-ovs-dpdk
  - configure-network
  - validate-network
  - install-os
  - ceph-hci-pre
  - configure-os
  - ssh-known-hosts
  - run-os
  - reboot-os
[fultonj@runcible nfv-ovs-dpdk-sriov-hci{nfvhci}]$ 
```